src/bin/genesis.rs

```diff
@@ -0,0 +1,68 @@
+use dotenv::dotenv;
+use ethers::types::U256;
+use kakarot_rpc::test_utils::katana::genesis::KatanaGenesisBuilder;
+use lazy_static::lazy_static;
+use reth_primitives::B256;
+use serde::Serialize;
+use serde_with::serde_as;
+use starknet::core::serde::unsigned_field_element::UfeHex;
+use starknet_crypto::FieldElement;
+use std::{
+    collections::HashMap,
+    env::var,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+
+lazy_static! {
+    static ref GENESIS_FOLDER_PATH: PathBuf = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join(".katana");
+    static ref KAKAROT_CONTRACTS_PATH: PathBuf =
+        Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join("lib/kakarot/build");
+    static ref INCORRECT_FELT: String = "INCORRECT".to_string();
+    static ref COINBASE_ADDRESS: FieldElement = FieldElement::from(0x12345u32);
```

Comment:
Can we add a TODO for this?

Reply:
to do what? I'm not too sure what you mean

Reply:
If you mean set the coinbase in the hive genesis, this is done in the `try_into_genesis_json` function.

Reply:
Why do we use this constant then?

Reply:
this here is for generating the Katana genesis. Hive will also use the KatanaGenesisBuilder which has an input to set the coinbase address. Since here we are generating a genesis for testing, we don't care about it.

Reply:
ah yes ok, but this isn't obvious, let's add this as a comment then